graphql-schemas/schema.yml

```diff
@@ -4378,3 +4380,18 @@ confs:
   fields:
   - { name: name, type: string, isRequired: true }
   - { name: query, type: string, isRequired: true }
+
+- name: Network_v1
+  datafile: /app-sre/network-1.yml
+  fields:
+  - { name: schema, type: string, isRequired: true }
+  - { name: path, type: string, isRequired: true }
+  - { name: name, type: string, isRequired: true }
+  - { name: description, type: string, isRequired: false }
+  - { name: inUseBy, type: NetworkInUseBy_v1 }
+  - { name: networkAddress, type: string, isRequired: true, isUnique: true }
+  - { name: region, type: string, isRequired: false }
+  - { name: parentNetwork, type: Network_v1, isRequired: false }
+
+- name: NetworkInUseBy_v1
+  fields:
```

Comment:
I think we are missing the fields here.

Reply:
> I think we are missing the fields here.

Are we? The other PR will add it. I don't want to add it here because I'm not adding it to the schema. Nothing uses this so it shouldn't be a problem?

Reply:
Yeah, but not adding anything under fields is failing PR check.

Reply:
Hmm, I'm not sure what to add for it.

Reply:
I added a placeholder field to hopefully make CI play nice. It will cause a merge conflict for #610, but hopefully it's not very painful to resolve.

Reply:
Good.